client/src/app/tabs/cloud-bpmn/__tests__/BpmnEditorSpec.js

```diff
@@ -565,129 +566,103 @@ describe('cloud-bpmn - <BpmnEditor>', function() {
     });
 
 
-    describe('#updatePropertiesPanelErrors', function() {
+    describe('linting plugin', function() {
 
-      let cache,
-          instance,
-          modeler;
+      it('should set errors', async function() {
 
-      beforeEach(async function() {
-        modeler = new BpmnModeler();
+        // given
+        const linting = [
+          {
+            id: 'foo',
+            message: 'Foo'
+          }
+        ];
 
-        cache = new Cache();
+        const { instance } = await renderEditor(diagramXML, {
+          linting
+        });
 
-        cache.add('editor', {
-          cached: {
-            modeler
-          },
-          __destroy: () => {}
+        const setErrorsSpy = spy(instance.getModeler().get('linting'), 'setErrors');
+
+        // when
+        const oldLinting = [
+          {
+            id: 'bar',
+            message: 'Bar'
+          }
+        ];
+
+        instance.componentDidUpdate({ ...instance.props, linting: oldLinting });
```

Comment:
I'd love us to not directly call component lifecycle methods, though I understand that with the existing infrastructure it may be too difficult. Note that `this.props` don't change for the component when we call it like this. Still, I suggest we add a todo to change this once we ditch enzyme:

```suggestion
// TODO @philippfromme: update props instead of calling lifecycle method directly
instance.componentDidUpdate({ ...instance.props, linting: oldLinting });
```

Reply:
I know that. Unfortunately, you can only simulate changing props on the root wrapper when using Enzyme. I couldn't find any other way to test this behavior.